terragrunt/aws/glue/etl.tf

```diff
@@ -158,8 +156,6 @@ resource "aws_glue_job" "bes_crm_salesforce" {
     "--enable-metrics"                   = "true"
     "--enable-observability-metrics"     = "true"
     "--job-language"                     = "python"
-    "--python-modules-installer-option"  = "-r"
-    "--additional-python-modules"        = "s3://${var.glue_bucket_name}/${aws_s3_object.bes_crm_salesforce_requirements.key}"
```

Comment:
so with pythonshell, this is not needed or not used as such https://docs.aws.amazon.com/glue/latest/dg/add-job-python.html